### DISCUSSION TITLE
output size limitation option in toTex for matrices

### DISCUSSION THREAD

**Discussion by nycos62** [josdejong/mathjs#3328] [Ideas]
Hello, 
is there a simple way to limit matrix rendered size in Tex ?
imagine you have a 1000x1000 matrix in memory but just want to output a tiny part for user to not blow up his screen
ex : setting max display column and rows to 5 on a 1000x1000 matrix should output something like this
![image](https://github.com/user-attachments/assets/8a329986-09ee-49b7-b12c-58a06156dcc6)
(ref https://tex.stackexchange.com/questions/416635/labeling-the-dimensions-of-a-matrix )

quick generating code :
math.parse(math.kron(math.range(1,100),math.range(1,100)).toString()).toTex();

**Comment by josdejong** [comment]
That is an interesting idea. I guess this should be a behind a configuration option that you can turn on/off.

**Reply by nycos62** [reply]
yes , the code to change seems to be there : 
mathjs/src/expression/node/ArrayNode.js line 155
